pandas/tests/extension/base/ops.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import numpy as np
+import pandas as pd
+from .base import BaseExtensionTests
+
+
+class BaseOpsTests(BaseExtensionTests):
+    """Various Series and DataFrame ops methos."""
+
+    def check_op(self, s, op, other, exc=NotImplementedError):
+
+        with pytest.raises(exc):
+            getattr(s, op)(other)
+
+    def test_arith_scalar(self, data, all_arithmetic_operators):
+        # scalar
+        op = all_arithmetic_operators
+        s = pd.Series(data)
+        self.check_op(s, op, 1, exc=TypeError)
+
+    def test_arith_array(self, data, all_arithmetic_operators):
+        # ndarray & other series
+        op = all_arithmetic_operators
+        s = pd.Series(data)
+        self.check_op(s, op, np.ones(len(s), dtype=s.dtype.type),
+                      exc=TypeError)
+
+    def test_divmod(self, data):
+        s = pd.Series(data)
+        self.check_op(s, divmod, 1, exc=TypeError)
+
+    def _compare_other(self, data, op, other):
+        s = pd.Series(data)
+
+        if op in '__eq__':
```

Comment:
in -> == ?

But it's not fully clear to me what you are testing here. You are testing that the equality operators are *not* implemented?

Reply:
This is from the @jreback implementation.  Let me study it in more detail and try it another way.

Reply:
@jorisvandenbossche So I studied this, and I think the intent here is to say that by default, the `ExtensionArray` `eq` and `ne` operators are not implemented, but that doing an `eq` or `ne` operation on a `Series` containing the `ExtensionArray` produces a result.

But I have made some changes to these tests to clarify that we are testing operator names versus operators.